### PR TITLE
Add posibility to recursively find test scripts in configured directory

### PR DESCRIPTION
**Pull request by Kozoriz**
- Added variable with name of directory with test scripts to config.py
- Updated way to find scripts in test directory 

Related task : [APPLINK-25934](https://adc.luxoft.com/jira/browse/APPLINK-25934)

### CODE REVIEW

**Comment by Kozoriz** [comment]
Please review : @OHerasym, @okozlovlux, @dev-gh, @VProdanov, @LevchenkoS, @anosach-luxoft , @LuxoftAKutsan

**Comment by okozlovlux** [comment]
@Kozoriz reviewed

**Comment by OHerasym** [comment]
@Kozoriz , feature into master?

**Comment by dev-gh** [comment]
Reviewed

**Comment by Kozoriz** [comment]
@OHerasym, yes, in this repo we have only master

**Comment by OHerasym** [comment]
Reviewed

**Comment by VProdanov** [comment]
Reviewed

**Comment by LevchenkoS** [comment]
@Kozoriz, reviewed

File: config.py [hunk 1]
```diff
@@ -45,6 +45,7 @@
 atf_build_dir = work_dir + "atf_build/"
 
 scripts_clone_dir = work_dir + "sdl_atf_test_scripts/"
+test_scripts_dir = scripts_clone_dir + "test_scripts/"
```
Comment (LuxoftAKutsan) [inline config.py]: @Kozoriz  please add comment with description of variable

**Comment by LuxoftAKutsan** [comment]
Reviewed

File: config.py (hunk 1 above)
Comment (Kozoriz) [inline config.py]: @LuxoftAKutsan done in 3d688ea

**Pull request merged**